script/teleporters.lua

```diff
@@ -206,8 +206,10 @@ local make_teleporter_gui = function(player, source)
 
   local sorted = {}
   local i = 1
+  local player_location = player.surface.name
   for name, teleporter in pairs (network) do
-    if teleporter.teleporter.valid then
+    local teleport_location = teleporter.teleporter.surface.planet.name
```

Comment:
This will crash if the teleporter is invalid, so it should go after the teleporter.valid check, it can be a nested `if`

Reply:
That should do it.